src/libraries/System.ComponentModel.Annotations/ref/System.ComponentModel.Annotations.cs

```diff
@@ -65,6 +65,8 @@ public sealed partial class CustomValidationAttribute : System.ComponentModel.Da
     {
         public CustomValidationAttribute([System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute(System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicMethods)] System.Type validatorType, string method) { }
         public string Method { get { throw null; } }
+        public override bool RequiresValidationContext { get { throw null; } }
+        public override object TypeId { get { throw null; } }
```

Comment:
CustomValidationAttribute is sealed, so these technically aren't necessary. Overrides need to be in the ref so that a derived type doing `base.Member` will call to the override rather than skipping it, but that's not an issue on sealed types.